### DISCUSSION TITLE
Normcap update checking

### DISCUSSION THREAD

**Discussion by Psycodex2001** [dynobo/normcap#284] [Q&A]
Does this tool app have any update checking feature and an update option? Or do I have to constantly check this on its GitHub repository or other app sources?

Anyways, thanks so much for the app creation and support!

**Comment by dynobo** [comment]
Yes, it has an update check feature. It's deactivated by default, but you can enable it via the menu:
![image](https://user-images.githubusercontent.com/11071876/198823919-50007279-d295-4df9-896e-b2d1703a78cd.png)

It frequently checks NormCap's GitHub Page for new releases (since v0.3.13 weekly, before on every launch). If it detects a new release, it shows a pop-up with a link to the download. (Auto-updating would be nice to have, but it's really complicated and not a high priority for me.)

Hint: For users which installed NormCap via a Package Manager (e.g. on Linux FlatPak or AUR) I recommend keeping the update check _disabled_ and rely on the package manager's update mechanism instead.
